### DISCUSSION TITLE
small help with Migration to v4

### DISCUSSION THREAD

**Discussion by VADDIJASWANT** [clientIO/joint#2564] [Q&A] [answered]
### Introduction

Hello everyone,

I am currently working on a project that uses ASP.NET MVC with JavaScript and jQuery on the front end. We have been using JointJS version 3.7 and everything was working fine. We included the `rappid.js` file, which we got from the license folder, to our scripts in the project.

Recently, JointJS version 4 was released and we decided to migrate to this new version. However, we encountered some issues during the migration. We tried to only add the `jointjs-plus.js` to the scripts but we are getting some build errors.

small note we are not using npm or yarn in our project, so we have to manually add the files to the project.

I was wondering if there is a specific path pattern and specific js files that we need to add when migrating to JointJS version 4? Any help or guidance would be greatly appreciated.

Thank you in advance!

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
Have you tried to use `build/packages/joint-plus.js` as we do in the _KitchenSink_ *JS* application?

**Reply by VADDIJASWANT** [reply]
In the beginning, I used the kitchensink application as a reference and added build/packages/joint-plus.js into my script folder directly. I think this led to the build errors which I got due to this build folder. However, following your advice from yesterday I have revisited the kitchensink application to cross check once and I completely eliminated the build and packages folders, retaining only Joint-plus.js. After integrating that into my application, everything is functioning smoothly. thanks a lot for help.
